shared/teams/team/container.js

```diff
@@ -94,7 +97,10 @@ const mapDispatchToProps = (
   dispatch: Dispatch,
   {navigateUp, newOpenTeamRole, setOpenTeamRole, setRouteState, routeProps}
 ): DispatchProps => ({
-  _loadTeam: teamname => dispatch(TeamsGen.createGetDetails({teamname})),
+  _loadTeam: teamname => {
+    dispatch(TeamsGen.createGetDetails({teamname}))
+    dispatch(TeamsGen.createGetTeamOperations({teamname}))
```

Comment:
I think this might be better if it's in getDetails, no? Is there a case where we want to get details, but not team operations?

Reply:
I'll make a pr